x-pack/test/security_solution_cypress/cypress/e2e/ai4dsoc/privileges/security_privileges.cy.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login } from '../../../tasks/login';
+import { visit } from '../../../tasks/navigation';
+
+const selectSpaces = (): void => {
+  cy.get('[data-test-subj="addSpacePrivilegeButton"]').click();
```

Comment:
It would be great if the selectors can be extracted to the `screens` folder,  and the overall action to `tasks`.

Reply:
Updated in e80ded6b30abdf04dc6ee9e24d71878e3d144f27 and in b4f59d618671abc7ac284878ad77eb6082e3e236